### PR TITLE
Feature #652 display authors

### PR DESCRIPTION
**Pull request by flbulgarelli**
First try. Please have a look @faloi 

screencapture-central-localmumuki-io-3000-lessons-25-fundamentos-practica-primeros-programas-1472580475357
 

Contributors are not yet listed.

### CODE REVIEW

**Comment by faloi** [comment]
Not bad, although in my mind was something more like this:
image

**Comment by flbulgarelli** [comment]
@faloi, right, but you the view you are proposing is not consistent with recent changes in bibliotheca and editor. We need other kind of structured information: name, image or github name/ linkedin profile, employeer, etc. I think personally it is out of current scope.  

Perhaps  you are proposing something like a non-free text in the authors fields, and using for example the email in combination with the linkedin API, are you?

**Comment by faloi** [comment]
I'm the salesman, so I'm not proposing anything. Just telling you how I imagine the view. :stuck_out_tongue:

Now seriously, I think structuring the information in the editor could be a good idea: free text is hard to parse if we want to do something with it later. 

About the view, it's good enough for now, except for one thing: I'd like a free text field right after the authors, to specify something like "based on X book and blabla".

**Comment by flbulgarelli** [comment]
> About the view, it's good enough for now, except for one thing: I'd like a free text field right after the authors, to specify something like "based on X book and blabla".

That is the contributors field, a free text.

Since it may be long, and I would not like to disturb the user experience with information that is not really important for the normal user, I would just add a link to it: "this guide was developed by....and [many other contributors](link)"

However, in any case, this information is not yet available. Waiting @fedescarpa to implement https://github.com/mumuki/mumuki-teacher-tools/issues/145 first. 

In the meanwhile, I can add a link to the github repository, and merge this PR, do you agree @faloi ?

**Comment by faloi** [comment]
Ok, merge it.

**Comment by flbulgarelli** [comment]
Second try @faloi 

screencapture-central-localmumuki-io-3000-exercises-105-fundamentos-practica-primeros-programas-calentando-motores-1473110967173
screencapture-central-localmumuki-io-3000-lessons-25-fundamentos-practica-primeros-programas-1473110987631

Now I am rendering it on the footer, and not only on the guide page but also on the exercise page. Texts are smaller and no unnecessary title is displayed

**Comment by flbulgarelli** [comment]
PS: Please notice that it is the same strategy followed by Wikipedia

**Comment by faloi** [comment]
Ohhh this disclaimer really can be seen.

Does the "many others" link work or is just a stub? :P

**Comment by flbulgarelli** [comment]
@faloi it works. It redirects to the raw COLLABORATORS.txt file in github.

It could redirect to another page, but... I don't really think it worths the effort. 

Shall I proceed with merge?

**Comment by faloi** [comment]
Yes sir, proceed.
